browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1588,9 +1588,12 @@ L.Control.JSDialogBuilder = L.Control.extend({
 	},
 
 	_editControl: function(parentContainer, data, builder, callback) {
-		var edit = L.DomUtil.create('input', 'ui-edit ' + builder.options.cssClass, parentContainer);
+		var container = L.DomUtil.create('div', 'ui-edit-container' + builder.options.cssClass, parentContainer);
```

Comment:
you need a space at the end of string, to not concatenate classes

Reply:
Ops sorry 🙇🏼